### PR TITLE
We appeared to lose the hover effect for the search icon

### PR DESCRIPTION
**Pull request by lol768**
This adjusts the selector so it'll work with `.fas`. It's a bit of a stopgap fix, I guess ideally we'd be using a selector unrelated to the icon classes.

### CODE REVIEW

File: less/search.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.id7-search {
+id7-search {
```
Comment (halfninja) [inline less/search.less]: Don't think you meant to drop the `.` here

File: less/search.less [hunk 2]
```diff
@@ -23,7 +23,7 @@
     .transition(color .15s linear);
   }
 
-  .twitter-typeahead:hover + .fa, .fa:hover {
+  .twitter-typeahead:hover + .fa, .fa:hover, .twitter-typeahead:hover + .fas, .fas:hover {
```
Comment (halfninja) [inline less/search.less]: Could completely replace the `.fa` versions - we don't need to support FA4 icons.

Alternatively we only ever use `<i>` in here for icons so could select on that instead.

Comment (lol768) [reply less/search.less]: Won't dropping `.fa` support stop SiteBuilder from deploying the shims w/o upgrading if it wants to?

Comment (halfninja) [reply less/search.less]: Yeah, but we'd expect Sitebuilder to upgrade its own icons to FA5 as part of this - the shims are just for user content.

Comment (lol768) [reply less/search.less]: Fair enough, I'll simplify this selector

**Pull request merged**